obspy/io/reftek/core.py

```diff
@@ -241,10 +241,14 @@ def to_stream(self, network="", location="", component_codes=None,
             else:
                 eh = EHPacket(et_packets[0])
             # only "C0" encoding supported right now
-            if eh.data_format != b"C0":
+            if eh.data_format == b"C0":
+                encoding = 'C0'
+            elif eh.data_format == b"C2":
+                encoding = 'C2'
```

Comment:
I agree, but..
 * I didn't want to do any unnecessary decoding of the encoding field of the packets array, but also
 * using a `kwarg` that expects a bytestring seemed also pretty strange to me

Open for suggestions.